azure-sfmesh-maven-plugin/pom.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+      <groupId>com.microsoft.azure</groupId>
+      <artifactId>azure-maven-plugins-pom</artifactId>
+      <version>1.0.1-SNAPSHOT</version>
+      <relativePath>../azure-maven-plugins-pom</relativePath>
+  </parent>
+
+  <groupId>com.microsoft.azure</groupId>
+  <artifactId>azure-sfmesh-maven-plugin</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
```

Comment:
1.0.0-SNAPSHOT is different from the sample snippet in readme "0.1.0", is it expected?

Reply:
There is already a initial version of plugin with version 0.1.0. So leaving it as 0.1.0.